### ISSUE TITLE
Radicals 

### ISSUE THREAD

**Issue by yakubog** [yakubog/vic3_community#1]
AI is deathspiralling into revolutions.

Possible fix: SOL_ADJUSTMENT_POP_IMPACT_FACTOR changed to 0.001.

**Comment by Tophattingson** [comment]
I do not believe that pops being unhappy when their standard of living decreases is a bug. It's a matter of balance. The current level seems appropriate enough. The death spiral revolutions that follow are caused by the AI being unable to manage it's budget from a combination of too large army, no consumption taxes, and being unable to declare bankruptcy, leading to the -50% throughput modifier from defaulting.

If countries are still too revolt-happy after fixing everything else we can fix, then it would be time to look at changing this number.
